### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by pvanderknyff**
- Add the extra requirements for packaging a connector into a Taco
- Add known issues for fully qualified link and the Mac error

### CODE REVIEW

**Comment by chrisjunlee** [comment]
Need more ASCII art.

File: README.md [hunk 1]
```diff
@@ -63,6 +68,14 @@ Many things might change, but the most likely changes are how a connector is pac
 
 [Visit the project website and documentation here.](https://tableau.github.io/connector-plugin-sdk/)
 
+# Known Issues
+
+**(Mac Only) Packaged Connectors (.taco files) throws unexpected error in 2019.4**
```
Comment (chrisjunlee) [inline README.md]: Add newline please

File: README.md [hunk 2]
```diff
@@ -63,6 +68,14 @@ Many things might change, but the most likely changes are how a connector is pac
...
+You can work around this by skipping signature verification with the command line argument `-DDisableVerifyConnectorPluginSignature=true`.
+
+**Support links that are not fully qualified throw error when clicked on**
```
Comment (chrisjunlee) [inline README.md]: newline

**Comment by sidwray** [comment]
Ship it!

**Pull request merged**